src/site/content/en/fast/chrome-ux-report-pagespeed-insights/index.md

```diff
@@ -99,23 +111,25 @@ user experiences across all pages of an origin. You can get the same stats for a
 closely aligned with what is available on [BigQuery](/chrome-ux-report-bigquery/), while the page-level
 performance is not made available to query.
 
-{% Img src="image/admin/7Xr7VVdZMbRlEMlgq3cj.png", alt="Origin CrUX performance in PageSpeed Insights", width="800", height="371" %}
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/Mer6ovkOuMxHZH6fhHY7.png", alt="Origin CrUX performance in PageSpeed Insights", width="800", height="426" %}
 
 There is one major difference between the origin-level data on PSI versus
 BigQuery. The datasets on BigQuery are released once a month and encompass the data from the previous calendar month. For example, the 202005 dataset
-includes all user experiences that occurred in May 2020. On the other
-hand, PSI aggregates new data every day encompassing the previous 28 days. So
+includes all user experiences that occurred in the 28 days up until the end of May 2020.
```

Comment:
I actually think we can skip over this implementation detail. The important thing to understand is the difference between trailing and monthly aggregation windows.
```suggestion
includes all user experiences that occurred in May 2020.
```